lib/qa/authorities/authority_with_sub_authority.rb

```diff
@@ -1,12 +1,18 @@
 module Qa::Authorities
-  class AuthorityWithSubAuthority < Base
```

Comment:
I'm not sure it's worth it. This'll be a new minor version as the major is less than 1.  I think the messages that are left give you a good clue as to what happened if you run into it.  I don't think we should work too hard to carry forward a bad design.